projects/NXP/devices/iMX8/bootloader/firmware

```diff
@@ -0,0 +1,9 @@
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
+
+[ -n "$ATF_PLATFORM" ] && cp -av $(get_install_dir atf)/usr/share/bootloader/bl31.bin .
+
+cp -av $(get_build_dir firmware-imx)/firmware/hdmi/cadence/signed_hdmi_imx8m.bin $(get_build_dir $BOOTLOADER)/
```

Comment:
Are you sure this is needed? AFAIK Blobs for HDMI are usually needed only for HDCP 2.2.

Reply:
Yes it's needed
```
WARNING 'signed_hdmi_imx8m.bin' not found, resulting binary is not-functional
  CFGS    spl/u-boot-spl.cfgout
  MKIMAGE flash.bin
./tools/mkimage: Can't open spl/u-boot-spl-ddr.bin: No such file or directory
make[1]: *** [arch/arm/mach-imx/Makefile:158: flash.bin] Error 1
```